### PR TITLE
sonumirror

### PR DESCRIPTION
**Pull request by Sunilkumarchoudari**
## Summary
Sonu Mirror - Reflect and Display the Daily Life info on Dressing Mirror.
I am Building this project for daily use to reflect and display the information in mirror

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by Sunilkumarchoudari** [comment]
This is looking Good

**Comment by Sunilkumarchoudari** [comment]
Approve the changes

**Review by Sunilkumarchoudari — COMMENTED** [review]
Looking Good Approve the changes.

**Review by Sunilkumarchoudari — COMMENTED** [review]
I am requetsing to for approval

**Review by Sunilkumarchoudari — COMMENTED** [review]
Approve the changes

**Review by Sunilkumarchoudari — COMMENTED** [review]
Now it is looking Good - Requesting for approval

**Review by Sunilkumarchoudari — COMMENTED** [review]
## Changes
Changes occured in list - as well as in plan .

**Review by Sunilkumarchoudari — COMMENTED** [review]
File Extension has changed to MD

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hello - this seems like a very nice project (actually kinda wanted to do it myself as well haha)

Can you please confirm you've budgeted for taxes & shipping?

Also, do you already have a power supply for your raspberry pi? Regular phone chargers don't provide enough power for it to rub properly

**Comment by Sunilkumarchoudari** [comment]
> Hello - this seems like a very nice project (actually kinda wanted to do it myself as well haha)
> 
> Can you please confirm you've budgeted for taxes & shipping?
> 
> Also, do you already have a power supply for your raspberry pi? Regular phone chargers don't provide enough power for it to rub properly

Thank you sir, i have charger somehow connector to connect with module. There is no change in budget & thank you for asking sir.

**Comment by Sunilkumarchoudari** [comment]
> Also, please fill out the PR template (just click edit in the first message)

Done. Thank you

**Review by Sunilkumarchoudari — COMMENTED** [review]
shipping charges added

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hello Sunil! Thanks for your proposal! Can you please fill out the Plan section of your request to include details on what steps you'll take to create your mirror? Maybe include any tutorials you'd be following or your inspiration? Thanks

**Review by Sunilkumarchoudari — COMMENTED** [review]
Plan-Upgraded

**Comment by Sunilkumarchoudari** [comment]
> Hello Sunil! Thanks for your proposal! Can you please fill out the Plan section of your request to include details on what steps you'll take to create your mirror? Maybe include any tutorials you'd be following or your inspiration? Thanks

Plan has been updated , Thank you.

File: sonumirror.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+Just want to clarify that these websites I'll be using to buy the material.
+
+## Budget
+
+| Product           | Supplier/Link                         | Cost    |
+| ---------------   | ------------------------------------- | ------  |
+| Rasberry Pi       | https://amzn.eu/d/h6dXBEs             | $176.44 |
+| Webcam            | https://amzn.eu/d/548ZtSv             | $23.20  |
+| Frame             | https://amzn.eu/d/548ZtSv             | $10.00  |
+| Connectors        |                                       | $5.00   |
```
Comment (cfanoulis) [inline sonumirror.md:77]: please clarify where you're buying the connectors from! if the shop doesn't have a website, please just note it's name

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hey, thanks for updaing your plan. Please include any links to guides you'll be following + address the following:
(if you're not following any guide pls say so!)

File: sonumirror.md (hunk 1 above)
Comment (Sunilkumarchoudari) [reply sonumirror.md:77]: I will purchase the connectors from local store, from Gujarati Galli Hyderabad, here we have huge stores to purchase all the electornics needed. Thank you.

Comment (Sunilkumarchoudari) [reply sonumirror.md:77]: > I will purchase the connectors from local store, from Gujarati Galli Hyderabad, here we have huge stores to purchase all the electornics needed. Thank you.

Hello sir did check the changes

**Comment by cfanoulis** [comment]
Hi, I'm not seeing a detailed plan on how you'll proceed with your project - only some explanation behind your selection of materials. 

Since you're not including any guides or inspiration, please include a detailed plan of how you'll be building your project

**Comment by Sunilkumarchoudari** [comment]
> Hi, I'm not seeing a detailed plan on how you'll proceed with your project - only some explanation behind your selection of materials.
> 
> Since you're not including any guides or inspiration, please include a detailed plan of how you'll be building your project

Sir Updated Plan alogn guide with reference

**Review by Sunilkumarchoudari — COMMENTED** [review]
Updated Project Guide along with reference

**Review by cfanoulis — CHANGES_REQUESTED** [review]
Hi, I need you to fix your frame link - it currently links to your webcam

Are you sure your monitor will fit inside?

**Comment by Sunilkumarchoudari** [comment]
> Hi, I need you to fix your frame link - it currently links to your webcam
> 
> Are you sure your monitor will fit inside?

Yes sir.

**Comment by cfanoulis** [comment]
(I still need you to fix your webcam/frame link)

**Comment by cfanoulis** [comment]
@Sunilkumarchoudari hello! It's been a while since you've responded to this PR - are you still working on getting this updated?

**Comment by cfanoulis** [comment]
Hi, it's been a while since we've had any activity in this proposal. Please reply to this PR in the next 24 hours, else we will be closing this PR

**Pull request closed without merging**